.golangci.yml

```diff
@@ -74,6 +74,10 @@ linters-settings:
         disabled: true
       - name: cognitive-complexity
         disabled: true
+      - name: comment-spacings
```

Comment:
https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#comment-spacings

Reply:
seems like you need a few pragmas: `go:generate and nolint`?

Reply:
yeah, these pragmas are needed, but this linter fails in code blocks like https://github.com/ObolNetwork/charon/blob/main/app/errors/errors.go#L32-L47. I think this is a bug, so i chose to disable it
<img width="967" alt="Screenshot 2023-07-24 at 13 50 30" src="https://github.com/ObolNetwork/charon/assets/43061995/081b7a4d-ccce-46b6-a46c-acc74ae1beda">

Reply:
ok cool, add a comment that it doesn't support the latest go spec comments when disabling it.